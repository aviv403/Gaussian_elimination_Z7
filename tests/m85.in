1 2 0 2 2 6 4 1 1 1 2 4 6
0 5 4 4 6 1 1 3 6 0 1 6 5
2 6 2 1 2 6 2 6 2 0 2 6 1
0 5 0 4 2 5 1 0 2 5 6 4 1
