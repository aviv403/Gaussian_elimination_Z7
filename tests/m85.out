1 0 0 0 2 5 0 0 3 1 4 2 2
0 1 0 0 2 3 0 5 6 5 3 0 0
0 0 1 0 1 6 0 6 1 4 4 4 1
0 0 0 1 5 1 2 6 0 2 3 1 2
